lab_2_keywords_cooccurrence/main.py

```diff
@@ -123,8 +223,41 @@ def extract_candidate_keyword_phrases_with_adjoining(candidate_keyword_phrases:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
-
+    if not (candidate_keyword_phrases and phrases):
+        return None
+    if not (isinstance(candidate_keyword_phrases, list) and isinstance(phrases, list)):
+        return None
+    for elem in candidate_keyword_phrases:
+        if not isinstance(elem, tuple):
+            return None
+    punct = """!"#$%&'()*+,-./:;<=>?@[]^_`{|}~¿—–⟨⟩«»…⋯‹›\\¡“”"""
```

Comment:
do you need this?

Reply:
no, i was fixing my code and created this variable
i found another way and pushed the changes a couple of minutes ago, so punct doesn't exist anymore in my code